code/__HELPERS/_lists.dm

```diff
@@ -541,39 +541,6 @@
 	. = pick_weight(list_to_pick)
 	list_to_pick[.]--
 
-/**
-* Picks n items from a list
-* e.g. pick_n(list_of_stuff, 10) would return a list of 10 items from the list, chosen randomly.
-*/
-proc/pick_n(list/list_to_pick, n)
-	if(!islist(list_to_pick) || !list_to_pick.len || n <= 0)
-		return list()
-
-	/// The final list that gets returned
-	var/list/result
-	/// Shuffling the list and picking the first n indices is faster in some cases
```

Comment:
where does this come from exactly?

Reply:
if you have a large n relative to the size of the list length, then your chances of picking the same thing repeatedly goes up with the random index method